--- /homes/leiqing/chatzilla/chatzilla.properties.zh-CN	2008-08-24 01:23:05.000000000 +0800
+++ chatzilla.properties.en-US	2008-08-24 01:22:14.000000000 +0800
@@ -24,6 +24,9 @@
 cmd.ban.params
 cmd.ban.help
 cmd.cancel.help
+cmd.ceip.label
+cmd.ceip.params
+cmd.ceip.help
 cmd.charset.params
 cmd.charset.help
 cmd.channel-motif.params
@@ -209,6 +212,8 @@
 cmd.hop.label
 cmd.hop.params
 cmd.hop.help
+cmd.idle-away.help
+cmd.idle-back.help
 cmd.reconnect.format
 cmd.reconnect.label
 cmd.reconnect.params
@@ -264,6 +269,8 @@
 cmd.join.help
 cmd.join-charset.params
 cmd.join-charset.help
+cmd.jump-to-anchor.params
+cmd.jump-to-anchor.help
 cmd.kick.format
 cmd.kick.label
 cmd.kick.params
@@ -278,6 +285,10 @@
 cmd.label-user.label
 cmd.label-user.params
 cmd.label-user.help
+cmd.label-user-multi.format
+cmd.label-user-multi.label
+cmd.label-user-multi.params
+cmd.label-user-multi.help
 cmd.leave.format
 cmd.leave.label
 cmd.leave.params
@@ -441,15 +452,20 @@
 msg.err.max.dispatch.depth
 msg.err.invalid.regex
 msg.err.invalid.pref
+msg.err.invalid.file
 msg.err.failure
 msg.err.scriptload
-msg.err.pluginapi
+msg.err.pluginapi.noid
+msg.err.pluginapi.faultyid
+msg.err.pluginapi.noenable
+msg.err.pluginapi.nodisable
 msg.err.invalid.scheme
 msg.err.item.not.found
 msg.err.unknown.pref
 msg.err.unknown.network
 msg.err.unknown.channel
 msg.err.unknown.user
+msg.err.unknown.command
 msg.err.unknown.stalk
 msg.err.unknown.motif
 msg.err.invalid.charset
@@ -478,6 +494,8 @@
 msg.err.invalid.mode
 msg.err.away.save
 msg.err.invalid.url
+msg.err.no.channel
+msg.err.no.idleservice
 msg.bug318419.warning
 msg.bug318419.error
 msg.need.oper.password
@@ -672,6 +690,7 @@
 msg.output.to
 msg.output.file
 msg.output.progress
+msg.output.cancel
 msg.logging.off
 msg.logging.on
 msg.logfile.closed
@@ -684,6 +703,9 @@
 msg.already.connected
 msg.enter.nick
 msg.network.connecting
+msg.jumpto.button
+msg.jumpto.err.nochan
+msg.jumpto.err.noanchor
 msg.banlist.item
 msg.banlist.button
 msg.banlist.end
@@ -832,6 +854,7 @@
 msg.nick.prompt
 msg.list.rerouted
 msg.list.end
+msg.list.chancount
 msg.who.end
 msg.who.match
 msg.connection.attempt
@@ -864,6 +887,7 @@
 msg.you.joined
 msg.someone.joined
 msg.you.left
+msg.you.left.reason
 msg.someone.left
 msg.someone.left.reason
 msg.youre.gone
@@ -871,9 +895,11 @@
 msg.mode.all
 msg.mode.changed
 msg.away.on
+msg.idle.away.on
 msg.away.off
 msg.away.prompt
 msg.away.default
+msg.away.idle.default
 msg.you.quit
 msg.someone.quit
 msg.unknown.ctcp
@@ -920,12 +946,14 @@
 msg.save.err.failed
 msg.save.fileexists
 msg.save.successful
-msg.inst1.msg1
-msg.inst1.msg2
-msg.inst1.msgRply1
-msg.inst1.msgRply2
-msg.inst1.command.yes
-msg.inst1.command.no
+msg.ceip.msg1
+msg.ceip.msg2
+msg.ceip.enabled
+msg.ceip.disabled
+msg.ceip.command.yes
+msg.ceip.command.no
+msg.ceip.upload.ok
+msg.ceip.upload.failed
 munger.mailto
 munger.link
 munger.channel-link
@@ -985,10 +1013,16 @@
 pref.autoRejoin.help
 pref.away.label
 pref.away.help
+pref.awayIdleTime.label
+pref.awayIdleTime.help
+pref.awayIdleMsg.label
+pref.awayIdleMsg.help
 pref.awayNick.label
 pref.awayNick.help
 pref.bugURL.label
 pref.bugURL.help
+pref.bugURL.comment.label
+pref.bugURL.comment.help
 pref.charset.label
 pref.charset.help
 pref.collapseMsgs.label
